File: aa/PYT/PRAC/NET/task4.py
#!/usr/bin/python3

no = int(input("Enter the no: "))


if no < 0: 
   print ("-ve number")
elif no == 0:
   print ("no is eq to zero")
else:
   print ("+ve number")
   for i in range(0,no+1):
       i *=i
       print (i)
       

File: srinivas/PL_FILES/TASKS/TASK/TA/D3/task4.pl
#!/usr/bin/perl -w
use strict;
#Listing 4.5. A program that reads input lines and concatenates them.

print "Enter your input type as empty to quit : ";
my $input = <STDIN>;	
chomp($input);

my $a = "";

while ($input ne "")	{
		$a.=$input;
		$input = <STDIN>;
		chomp($input);
}

print"$a\n";




